pyproject.toml

```diff
@@ -33,9 +33,9 @@ classifiers=[
 ]
 dependencies = [
   "xrft>=0.3.0",
-  "xarray>=0.19.0,~=2022.3.0",
+  "xarray>=0.19.0,~=2022.6.0",
```

Comment:
What about removing the `2022.6`, since it now accepts it? We don't know yet if we can't run with 2023. We could use something like:
`xarray>=0.19.0`